Auto-register (#142)
dbos-inc/dbos-docs: docs/explanations/application-structure-explanation.md

All options are documented in our [configuration reference](../api-reference/configuration).

`src/operations.ts` is the _entrypoint_, where DBOS looks for your code.

> At startup, the DBOS runtime automatically loads all classes exported from this file, serving their endpoints and registering their decorated functions and dependencies.

Reviewer: ```suggestion
At startup, the DBOS runtime automatically loads all classes that are exported or (directly and indirectly) referenced from this file, serving their endpoints and registering their decorated functions.
```

export { OperationClass1, OperationClass2 } from './FileA';
export { OperationClass3 } from './operations/FileB';
```
It is not necessary to export classes that are already referenced by the entrypoint file(s), as these will be loaded and decorated methods will be registered.

> You can configure a list of entrypoints using the `runtimeConfig` section of the [configuration](../api-reference/configuration#runtime).

Reviewer 2: ```suggestion
You can also define multiple entrypoint files using the `runtimeConfig` section of the [configuration](../api-reference/configuration#runtime).
```
